lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -133,34 +142,32 @@ def configure(conf)
           fully_qualified_zone = fetch_gce_metadata('instance/zone')
           @zone = fully_qualified_zone.rpartition('/')[2]
         end
-        if @vm_id.nil?
-          @vm_id = fetch_gce_metadata('instance/id')
-        end
+        @vm_id = fetch_gce_metadata('instance/id') if @vm_id.nil?
```

Comment:
yes, all suggestions were from rubocop, based on this style guide: https://github.com/bbatsov/ruby-style-guide

like I said, I don't really agree with them, but the other projects I looked at do not make any alterations to the style guide (aside from one of them using 120 char lines) so I decided it was best to stick to the style guide strictly even where I don't agree.